networkx/algorithms/tests/test_simple_paths.py

```diff
@@ -112,6 +112,57 @@ def test_digraph_all_simple_paths_with_two_targets_cutoff():
     assert set(tuple(p) for p in paths) == {(0, 1, 2, 3), (0, 1, 2, 4)}
 
 
+def test_all_simple_paths_weighted_graph_with_weight_cutoff():
+    edges = [(1, 2), (2, 10), (1, 5), (5, 4), (4, 3), (3, 10)]
+    distances = [5, 6, 1, 3, 2, 2]
+    d = {e: {'Distance': dist} for e, dist in zip(edges, distances)}
+
+    G = nx.Graph(edges)
+    nx.set_edge_attributes(G, d)
+
+    paths = list(nx.all_simple_paths(G, 1, 10, cutoff={'Distance': 10}))
+    assert len(paths), 1
```

Comment:
Do you want to check if `len(paths)` is `1`?  If so, this should be `assert len(paths) == 1`.  You may want to review this:
http://doc.pytest.org/en/latest/assert.html

Reply:
I think that was the nose way to assert stuff. Thanks for the link